DevVehicle35-Motors/Models/MiniBus.cs

```diff
@@ -35,27 +36,27 @@ public MiniBus()
         }
         public decimal GetPrice()
```

Comment:
Maybe this method isn't necesary because this property already has a public get method on line 11. Therefore you can use that and avoid to rewrite this one.

Reply:
Ok, I changed that, Ty